src/main/java/com/marklogic/kafka/connect/DefaultDatabaseClientConfigBuilder.java

```diff
@@ -113,7 +113,7 @@ private SSLContext buildTwoWaySslContext(String certFile, String certPassword, M
 
         KeyManagerFactory keyManagerFactory;
         SSLContext sslContext;
-        try (InputStream keystoreInputStream = new FileInputStream(certFile)) {
+        try (InputStream keystoreInputStream = Files.newInputStream(Paths.get(certFile))) {
```

Comment:
Do you know if this is fixing a bug, or just adopting a better practice for accessing the file?

Reply:
I changed it because Jetbrains and Sonar recommended the change. Apparently FileInputStream implements a "finalize" method that can cause issues with garbage collection.